husky_control/CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ catkin_package()
 
 roslaunch_add_file_check(launch)
 
-catkin_install_python(PROGRAMS scripts/stow_ur5 scripts/stow_ur5_left scripts/stow_ur5_right DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+install(PROGRAMS scripts/stow_ur5 scripts/stow_ur5_left scripts/stow_ur5_right DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
Yes, it did work with Gazebo. Haven't tested it on the real robot as I haven't pushed this bunch of changes onto the robot yet (bar the description.xacro one), and we aren't using catkin-tools on the robot yet.